sass/components/_modeRules.scss

```diff
@@ -0,0 +1,39 @@
+.modeRules {
+  display: block;
+  height: auto;
+  width: auto;
+  padding: 20px 45px;
+  border-radius: 16px;
+  border: none;
+  background-color: $color-bgc;
+  text-align: center;
+  font-family: $font-family--primary;
+  font-style: normal;
+  font-weight: 500;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  //margin-top: 40px;
```

Comment:
Czy ta zakomentowana właściwość jest nadal potrzebna?

Reply:
Dodałam to, żeby odsunąć całe to okno od nazwy trybu, ale w sumie to chyba wymagania do innego taska. Usunę przy następnym commicie :)